src/Protocol/Models/Position.cs

```diff
@@ -12,20 +12,20 @@ public Position()
 
         }
 
-        public Position(long line, long character)
+        public Position(int line, int character)
         {
             Line = line;
             Character = character;
         }
         /// <summary>
         /// Line position in a document (zero-based).
         /// </summary>
-        public long Line { get; set; }
+        public int Line { get; set; }
 
         /// <summary>
         /// Character offset on a line in a document (zero-based).
         /// </summary>
-        public long Character { get; set; }
+        public int Character { get; set; }
```

Comment:
any reason for changing all the `long`s to `int`s?

Reply:
To make it easier to deal with the math when generating semantic tokens, otherwise a lot of potential conversions can happen.  I can try and roll it back if it is needed for something.  I dunno how many files have 2GB characters on a single line.

Reply:
PowerShell uses ints so I'll be happy to remove a bunch of `(int)`s all over the place :)